packages/dds/tree/src/test/testUtilsTests.spec.ts

```diff
@@ -62,5 +66,23 @@ describe("Test utils", () => {
 				{ type: "foo", fields: { f: [withHandleExpected] } },
 			]);
 		});
+
+		it("createSnapshotCompressor", () => {
```

Comment:
this test used to fail, with the ids it gets being non-deterministic. The changes to createAlwaysFinalizedIdCompressor were required to fix it.